modules/standard/Random.chpl

```diff
@@ -590,15 +602,15 @@ module Random {
       which has 64 bits of state and 32 bits of output.
 
       While the PCG RNG used here is believed to have good statistical
-      properties, it is not suitable for cryptographic use. Additionally,
-      if statistical properties of the random numbers are very important,
-      another strategy may be required.
+      properties, it is not suitable for generating key material for encryption.
+      Additionally, if statistical properties of the random numbers are very
+      important, another strategy may be required.
 
-      We have verified that the random numbers generated by this class can match
-      those generated by the C PCG reference implementation. However, this
-      implementation differs from the C PCG reference implementation in how it
-      produces random integers in a range and in support for generating random
-      `real` s.
+      We have good confidence that the random numbers generated by this class
+      match the C PCG reference implementation and have specifically verified
+      equal output given the same seed. However, this implementation differs
+      from the C PCG reference implementation in how it produces random integers
+      in a range and in support for generating random `real` s.
```

Comment:
This is a nice change overall.  Naming the routine used to generate bounded integers might help clarify this point.  Specifically "random integers in a range" is easily confused with Chapel's notion of "range"